sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -1256,14 +1257,24 @@ def _generate_delete_blobs_options(self,
                     if_tags_match_condition=if_tags_match_condition
                 )
 
-            query_parameters, header_parameters = self._generate_delete_blobs_subrequest_options(**options)
+            built_url = "/{}/{}{}".format(quote(container_name), quote(blob_name, safe='/~'), self._query_str)
```

Comment:
Needed to pull this out from how the `HttpRequest` was built previously since we need the URL to pass into `build_delete_request`